### DISCUSSION TITLE
DNS issue when accessing site hosted on Vercel DNS_PROBE_FINISHED_NXDOMAIN

### DISCUSSION THREAD

**Discussion by kerbelp** [vercel/community#2496] [Help]
### Summary

I'm getting DNS_PROBE_FINISHED_NXDOMAIN when accessing https://ai-collection.org/
Is there a DNS issue with Vercel?
Thanks.

**Comment by kerbelp** [comment]
Works now. Issue was probably fixed.

**Reply by OmarTheProgrammerr** [reply]
what did you do to fix it? I'm facing the same error now :/

**Reply by kerbelp** [reply]
I'm facing it too now. It's an internal issue on Vercel. I hope they will resolve it soon.

**Reply by OmarTheProgrammerr** [reply]
Oh I actually solved it. My problem was simply a conflict in the DNS records type A. I had multiple A records on domain.com ( where I bought my domain ) and they were pointing a different ip address than the vercel one, so I just went a head and deleted all of them and it worked!
